### ISSUE TITLE
Nightwatch output

### ISSUE THREAD

**Issue by adarshannaiynar** [mucsi96/nightwatch-cucumber#284]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report
[x] feature request
[ ] support request => Please do not submit support request here, instead use [Stack Overflow](https://stackoverflow.com/questions/tagged/nightwatch.js+cucumber)
```

**Current behavior**
The output of current execution is dots
![image](https://user-images.githubusercontent.com/24566775/30689462-a7575ae0-9ede-11e7-8432-1a435d20949c.png)

**Expected/desired behavior**
I would like to get the output of nightwatch. Is this possible with the current version

**What is the motivation / use case for changing the behavior?**
To get more clarity on the running tests on the console rather than as a report.

**Comment by mucsi96** [comment]
Hi! That would be nice! I have an idea how to solve it. Stay tuned 😀

**Comment by adarshannaiynar** [comment]
Glad to hear that! 😄

**Comment by mucsi96** [comment]
@adarshannaiynar its released in `8.2.0`

![image](https://user-images.githubusercontent.com/3163392/30774165-718eaeb2-a07e-11e7-981a-0686d059448f.png)

We can also improve more on that if you have any ideas.

**Comment by adarshannaiynar** [comment]
Thanks a lot. This was the thing I was looking for. Keep up the great work. 😄
